andela/hc-ravens
# Ft running too often 156687758

#### What does this PR do?
- Adds feature to check if job is running too often. It flags with an orange icon if a job runs earlier than expected with the reverse grace period consideration

#### Description of Task to be completed?
Have the application identify a job if it is running too often. Alert the user with an email that the job is in that inconsistent mode and add an icon to the users checks for each job that is in that state

#### How should this be manually tested?
- Login at [hc-ravens](http://hc-ravens.herokuapp.com)
- Add a check and copy the ping URL
- Run the URL on your browser address bar to activate the Check
- Run it once more before the timeout period elapses
- Visit [your checks](http://hc-ravens.herokuapp.com/checks/) and you should see an orange bell indicating 'Too often' a job.

#### Any background context you want to provide?
None 

#### What are the relevant pivotal tracker stories?
#156687758
